yaml_edit/lib/src/editor.dart

```diff
@@ -197,9 +199,7 @@ class YamlEditor {
   /// print(newNode.value); // "YAML"
   /// print(node.value); // "YAML Ain't Markup Language"
   /// ```
-  YamlNode parseAt(Iterable<Object> path, {YamlNode Function() orElse}) {
-    ArgumentError.checkNotNull(path, 'path');
-
+  YamlNode parseAt(Iterable<Object?> path, {YamlNode Function()? orElse}) {
```

Comment:
`parseAt()` now returns a non-nullable `YamlNode` and orElse must return the same.